src/OpenColorIO/GpuShaderClassWrapper.h

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
```

Comment:
[Minor] Should add include guards here.

Reply:
@remia  good catch! include guards added.